systest/cluster/cluster.go

```diff
@@ -517,6 +569,34 @@ func (c *Cluster) AddSmeshers(tctx *testcontext.Context, n int, opts ...Deployme
 	return nil
 }
 
+func (c *Cluster) AddRemoteSmeshers(tctx *testcontext.Context, n int, opts ...DeploymentOpt) error {
+	if err := c.resourceControl(tctx, n); err != nil {
+		return err
+	}
+	if err := c.persist(tctx); err != nil {
+		return err
+	}
+	flags := maps.Values(c.smesherFlags)
+	endpoints, err := extractP2PEndpoints(tctx, c.clients[:c.bootnodes])
+	if err != nil {
+		return fmt.Errorf("extracting p2p endpoints %w", err)
+	}
+	dopts := []DeploymentOpt{WithFlags(flags...), WithFlags(Bootnodes(endpoints...), StartSmeshing(false))}
+	dopts = append(dopts, opts...)
+	clients, err := deployRemoteNodes(tctx, c.nextSmesher(), c.nextSmesher()+n, c.GoldenATX(), dopts...)
+	if err != nil {
+		return err
+	}
+	bootnodes := c.clients[:c.bootnodes]
+	smeshers := c.clients[c.bootnodes:]
+	c.clients = nil
+	c.clients = append(c.clients, bootnodes...)
+	c.clients = append(c.clients, smeshers...)
+	c.clients = append(c.clients, clients...)
+	c.smeshers += len(clients)
+	return nil
```

Comment:
I think this is the same:
```suggestion
	c.clients = append(c.clients, clients...)
	c.smeshers += len(clients)
	return nil
```

Reply:
Yeah probably, I copied this from `Addsmeshers` without thinking much about it 😅